template-processors/base/bin/wrapper.sh

```diff
@@ -17,9 +17,18 @@
 set -euxo pipefail
 
 export HOME=/tmp
-/usr/local/bin/gitClone.sh
-/usr/local/bin/discoverEnvironment.sh
-source $HOME/envs.sh
-/usr/local/bin/processParameters.sh
-/usr/local/bin/processTemplates.sh
-/usr/local/bin/resourceManager.sh
+
+case "$ACTION" in
+  create)
+    /usr/local/bin/gitClone.sh
+    /usr/local/bin/discoverEnvironment.sh
+    source $HOME/envs.sh
+    /usr/local/bin/processParameters.sh
+    /usr/local/bin/processTemplates.sh
+    /usr/local/bin/resourceManager.sh
+    ;;
+  delete)
+    /usr/local/bin/discoverEnvironment.sh
+    /usr/local/bin/resourceManager.sh
+    ;;
+esac
```

Comment:
Before, when there's been an unsuccessful hello-world application deployment and then one decided to delete the gitopsconfig, the command didn't return (because deletion job was failing on gitClone.sh, processParameters.sh, or processTemplates.sh - the finalizer was never removed, so k8s garbage collector couldn't remove the gitopsconfig).

Before:
<details><summary><code>time kubectl delete gitopsconfigs.eunomia.kohls.io --all -n eunomia-hello-world-yaml-demo</code></summary><br><pre>
$ time kubectl delete gitopsconfigs.eunomia.kohls.io --all -n eunomia-hello-world-yaml-demo 
gitopsconfig.eunomia.kohls.io "hello-world-yaml" deleted
^C
real    9m48.133s
user    0m0.188s
sys     0m0.195s
</pre></details>

Now:
<details><summary><code>time kubectl delete gitopsconfigs.eunomia.kohls.io --all -n eunomia-hello-world-yaml-demo</code></summary><br><pre>
$ time kubectl delete gitopsconfigs.eunomia.kohls.io --all -n eunomia-hello-world-yaml-demo 
gitopsconfig.eunomia.kohls.io "hello-world-yaml" deleted

real    0m12.661s
user    0m0.107s
sys     0m0.117s
</pre></details>